packages/terra-form-field/src/Field.jsx

```diff
@@ -31,6 +32,10 @@ const propTypes = {
    * The htmlFor attribute on the field label.
    */
   htmlFor: PropTypes.string,
+  /**
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the `@private` syntax so this doesn't show up in the generated prop-types documentation.